test/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/triage/SingleDisposalsTriageControllerSpec.scala

```diff
@@ -5794,15 +5809,23 @@ class SingleDisposalsTriageControllerSpec
     representeeAnswers: Option[RepresenteeAnswers] = None,
     isAmend: Boolean = false
   ): Unit = {
-    val draftReturn =
-      sample[DraftSingleDisposalReturn]
-        .copy(triageAnswers = currentAnswers, representeeAnswers = representeeAnswers, gainOrLossAfterReliefs = None)
+    val draftReturn = sample[DraftSingleDisposalReturn].copy(
+      triageAnswers = currentAnswers,
+      representeeAnswers = representeeAnswers,
+      gainOrLossAfterReliefs = None
+    )
 
-    val fillingOutReturn =
-      sample[FillingOutReturn].copy(
-        draftReturn = draftReturn,
-        amendReturnData = if (isAmend) Some(sample[AmendReturnData]) else None
-      )
+    val fillingOutReturn = sample[FillingOutReturn].copy(
+      draftReturn = draftReturn,
+      amendReturnData =
+        if (isAmend)
+          Some(
+            sample[AmendReturnData].copy(
+              shouldDisplayGainOrLossAfterReliefs = true
```

Comment:
this also prevents testing of the `false to `true` behaviour - I'd leave it as it was and make sure the `updateJourney` you supply to the function takes care of setting that flag correctly